docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,133 @@
+
+Prior to the actual swap execution there is usually a [Negotiation](negotiation.md) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice generates a secret and hashes it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times on the for the [HTLC](#htlc)s.
+1. Bob receives the swap proposed by Alice and sends back:
+    1. His refund address on Bitcoin.
+    1. His redeem address on Ethereum.
+1. Alice has to fund first, she deploys an Ethereum HTLC transferring the agreed amount of ETH into the contract.
+1. Bob monitors the Ethereum ledger and notices that Alice has deployed the HTLC funding the ETH. 
+He now sends a Bitcoin transaction into the Bitcoin network that is guarded by the script-hash of the Bitcoin HTLC (Bitcoin script).
+1. Alice monitors the Bitcoin ledger and notices that Bob has sent the funding transaction. She can now redeem the Bitcoin by spending the Bitcoin by revealing the secret.
+1. Bob is monitoring the Bitcoin ledger and notices that Alice has spent from the funding transaction by revealing the secret.
+He extracts the actual secret from Alice's redeem transaction and uses it to redeem on Ethereum. 
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vRjJeto_eQaZ8k5Ct00UzceHaSs-uIuvPmTFV6jP9SP6N-EKmct8H4cfN3xa2h-RFs8Gm4Gia41Je7m/embed?start=true&loop=false&delayms=1000" 
+    frameborder="0" 
+    width="801" 
+    height="480" 
+    allowfullscreen="true"
+    mozallowfullscreen="true" 
+    webkitallowfullscreen="true">
+</iframe>
+
+The COMIT protocol acts as enabler for atomic swaps in applications.
+The [COMIT network daemon (cnd)](../comit-protocol/comit-protocol-stack.md#comit-network-daemon-cnd) mainly does two things:
+
+1. Monitor the ledgers by processing blocks supplied by blockchain nodes.
+1. Define the transaction details and hand them over to an application on top to send the actual signed transaction into the respective blockchain network.
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vTSToxZxKhOjfUWHVL5sjjjyGTdEXubjM3TpOpK-qR5Cjs7b6Tda9ZoX6n_NdM9iqgXBGOtPcPnjHAA/embed?start=false&loop=false&delayms=1000"
+    frameborder="0"
+    width="801"
+    height="480"
+    allowfullscreen="true"
+    mozallowfullscreen="true"
+    webkitallowfullscreen="true">
+</iframe>
+
+## Atomic Swap
+
+Atomic Swaps define a peer to peer exchange of two digital assets, where both parties are in control of their private keys throughout the whole execution of the swap.
+
+The atomicity of the swap is achieved through using [HTLC](#htlc)s as a programmable escrow.
```

Comment:
This reads like HTLCs are the _only_ way to do that.

Reply:
Perhaps

In the case of COMIT, the atomicity of the swap is achieved ...